Library/Homebrew/os/mac/linkage_checker.rb

```diff
@@ -105,6 +126,9 @@ def display_test_output
     display_items "System libraries", @system_dylibs if OS.linux?
     display_items "Missing libraries", @broken_dylibs
     puts "No broken dylib links" if @broken_dylibs.empty?
+    return unless OS.linux?
+    display_items "Unwanted system libraries", @bad_system_dylibs
```

Comment:
I'd prefer that personally, but I wanted to keep consistency with "Missing libraries".